docs/api/tests.foreman.cli.rst

```diff
@@ -223,11 +223,6 @@
 
 .. automodule:: tests.foreman.cli.test_syncplan
 
-:mod:`tests.foreman.cli.test_system_registration`
--------------------------------------------------
-
-.. automodule:: tests.foreman.cli.test_system_registration
-
```

Comment:
Being removed as now `tests.foreman.cli.test_system_registration` is renamed as `tests.foreman.cli.test_contenthost`.